src/tui/rendering.rs

```diff
@@ -66,6 +66,19 @@ impl Tui {
         }
     }
 
+    pub fn select_first_row(&mut self) {
+        self.select_row_by_index(Some(0));
```

Comment:
How this will behave when there is no processes displayed and you move to the first row?